### DISCUSSION TITLE
Saleor APP on Contentful CMS

### DISCUSSION THREAD

**Discussion by albertojuniorhc** [saleor/saleor#14784] [General]
Hi!

Is anyone able to use the Saleor APP on the Contentful CMS platform?

On Contentful, the APP installation looks pretty straight forward, we need to add just the Store API URL (https://my-project.saleor.cloud/graphql/), but when we try to select a product, I'm always getting the error message: 'There was an error fetching the product list.' 

<img width="621" alt="Screenshot 2023-11-16 at 12 43 53 PM" src="https://github.com/saleor/saleor/assets/13555301/92b6aeb7-e723-4198-9b6e-077a6ae3df6a">

Thanks!

**Comment by albertojuniorhc** [comment]
I found the issue, but I don't know how to solve it.
```JSON
{
    "errors": [
        {
            "message": "More than one channel exists. Specify which channel to use.",
            "locations": [
                {
                    "line": 2,
                    "column": 3
                }
            ],
            "path": [
                "products"
            ],
            "extensions": {
                "exception": {
                    "code": "GraphQLError"
                }
            }
        }
    ],
    "data": {
        "products": null
    },
    "extensions": {
        "cost": {
            "requestedQueryCost": 300,
            "maximumAvailable": 50000
        }
    }
}
```

How can I specify the channel in the APP, since the only config that I have is the API URL?

**Comment by maxkraanen** [comment]
@albertojuniorhc I had the same problem. At the moment you can only use the integration if you have only one channel in Saleor because you have to pass in the channel as argument and that's not possible. You can manage the channels in Saleor at /dashboard/channels

**Reply by albertojuniorhc** [reply]
Got it!
I started to investigate this APP from the docs: https://docs.saleor.io/docs/3.x/developer/app-store/apps/cms
Apparently, is an integration based on variant events, and creates the Saleor products as a content entry on contentful...

**Reply by albertojuniorhc** [reply]
And also supports the multi-channel...
